core/App/screens/CommonDecline.tsx

```diff
@@ -90,11 +90,13 @@ const CommonDecline: React.FC<CommonDeclineProps> = ({ navigation, route }) => {
 
       if (agent && credential) {
         await agent.credentials.declineOffer(credential.id)
+        agent?.credentials.sendProblemReport({ credentialRecordId: proof.id, message: t('CredentialOffer.Declined') })
         return
       }
 
       if (agent && proof) {
         await agent.proofs.declineRequest(proof.id)
+        agent?.proofs.sendProblemReport(proof.id, t('ProofRequest.Declined')) // currently, fails for connectionless case
```

Comment:
I'm not sure if this is true but I seem to recall us not wanting to send problem reports for declined creds/proofs for some reason. I'll double check and get back to you.

Reply:
It just looks strange now when Verifier sends Reqeust which always stays in `Requested` state even if Prover rejected it. Moreover, according to the RFC Prover should send Problem Report when rejects a request.